### PR TITLE
Stop requiring pyOpenSSL.

### PR DESCRIPTION
**Pull request by rimey**
This is no longer needed.

### CODE REVIEW

**Comment by dhermes** [comment]
Good catch! #1518 (combined with https://github.com/google/oauth2client/pull/395 and https://github.com/google/oauth2client/pull/396) made it so we don't need this any longer.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -64,22 +64,6 @@ repo, from which you can submit a pull request.
...
    $ $VENV/bin/python setup.py develop
 
-I'm getting weird errors... Can you help?
------------------------------------------
-
-Chances are you have some dependency problems...
-If you're on Ubuntu,
-try installing the pre-compiled packages::
-
-  $ sudo apt-get install python-crypto python-openssl libffi-dev
```
Comment (dhermes) [inline CONTRIBUTING.rst]: I think `libffi-dev` might still be needed?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -64,22 +64,6 @@ repo, from which you can submit a pull request.
...
-
-or try installing the development packages
-(that have the header files included)
-and then ``pip install`` the dependencies again::
-
-  $ sudo apt-get install python-dev libssl-dev libffi-dev
```
Comment (dhermes) [inline CONTRIBUTING.rst]: `python-dev` will certainly be helpful

File: CONTRIBUTING.rst (hunk 1 above)
Comment (rimey) [inline CONTRIBUTING.rst]: It didn't come up in the testing I did.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (rimey) [inline CONTRIBUTING.rst]: This is `CONTRIBUTING`. Anybody building the library from sources should be able to figure out the error about `Python.h` being missing. The crypto dependencies were much nastier.

Comment (dhermes) [inline CONTRIBUTING.rst]: I think you may be overestimating. It's very common for people to not know what they're saying when native extensions don't build due to missing `Python.h`.

Comment (rimey) [inline CONTRIBUTING.rst]: PTAL

**Comment by dhermes** [comment]
LGTM.

@tseaver PTAL?

**Comment by tseaver** [comment]
Thanks, @rimey!

**Pull request merged**